data/api/client-server/report_content.yaml

```
@@ -29,11 +29,20 @@ securityDefinitions:
 paths:
   "/rooms/{roomId}/report/{eventId}":
     post:
-      summary: Reports an event as inappropriate.
+      summary: Reports an event as inappropriate. You must be joined to the room in order
+      to report an event.
       description: |-
         Reports an event as inappropriate to the server, which may then notify
         the appropriate people.
+        
+        It may be possible for clients to deduce whether an event exists by
```

nit: we're trying to not use "may" due to English not translating it nicely.
```suggestion
        It might be possible for clients to deduce whether an event exists by
```